src/bot/__tests__/requestReview.test.ts

```diff
@@ -0,0 +1,198 @@
+import { ShortcutParam } from '@/slackTypes';
+import { BOT_ICON_URL, BOT_USERNAME } from '@bot/constants';
+import { ActionId, Deadline, Interaction } from '@bot/enums';
+import { requestReview } from '@bot/requestReview';
+import { languageRepo } from '@repos/languageRepo';
+import { App } from '@slack/bolt';
+import { buildMockShortcutParam } from '@utils/slackMocks';
+
+describe('requestReview', () => {
+  describe('setup', () => {
+    let app: App;
+    const boundShortcutMethod = jest.fn();
+    const boundCallbackMethod = jest.fn();
+
+    beforeEach(() => {
+      app = {
+        shortcut: jest.fn() as any,
+        view: jest.fn() as any,
+      } as App;
+      requestReview.shortcut.bind = jest.fn().mockReturnValueOnce(boundShortcutMethod);
+      requestReview.callback.bind = jest.fn().mockReturnValueOnce(boundCallbackMethod);
```

Comment:
I'm not sure if it's really necessary. You could add a test that verifies:
```typescript
it('should bind this to shortcut and callback', () => {
  expect(requestReview.shortcut.bind).toBeCalledWith(app);
  expect(requestReview.callback.bind).toBeCalledWith(app);
});
```

Reply:
That's kinda what [these tests](https://github.com/sourceallies/hacker-rank-queue/pull/16/files/b0257d510f56b321f7e3c4a9c9faa384ca3acd00#diff-c54b611a97afcfbeb2caedfd0378ed87af2d720ee0c350635d24846938f5ba94R26-R32) do, though it's not super explicit

Reply:
I don't think those verify that the `this` is bound do they?
Again, that may be overkill.